laser_slam/src/benchmarker.cpp

```diff
@@ -0,0 +1,107 @@
+
+#include <chrono>
+#include <cmath>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <mutex>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <glog/logging.h>
+
+#include "laser_slam/benchmarker.hpp"
```

Comment:
Put that include at the top eg: https://github.com/ethz-asl/segmatch/blob/master/segmatch/src/segmatch.cpp#L1